lib/sass.dart

```diff
@@ -33,3 +33,23 @@ String render(String path,
       evaluate(sassTree, color: color, packageResolver: packageResolver);
   return toCss(cssTree);
 }
+
+/// Loads [source] as Sass, converts it to CSS, and returns the result.
```

Comment:
Done. I think the option will be addressed wth the indentedSyntax issue, https://github.com/sass/dart-sass/issues/7.